<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveUserConfirmationDialog</class>
 <widget class="QDialog" name="RemoveUserConfirmationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Confirm recipient's username</string>
         </property>
         <property name="buddy">
          <cstring>recipient_username_line</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="recipient_username_line"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Reason for deletion</string>
         </property>
         <property name="buddy">
          <cstring>deletion_reason_line</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="deletion_reason_line"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Confirm your password</string>
         </property>
         <property name="buddy">
          <cstring>confirm_password_line</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="confirm_password_line">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delete_button">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>recipient_username_line</tabstop>
  <tabstop>deletion_reason_line</tabstop>
  <tabstop>confirm_password_line</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
